Instagram/Models/Post.m

```diff
@@ -0,0 +1,55 @@
+//
+//  Post.m
+//  Instagram
+//
+//  Created by Sarah Wang on 7/7/21.
+//
+
+
+#import "Post.h"
+
+@implementation Post
+
+@dynamic postID;
+@dynamic userID;
+@dynamic author;
+@dynamic caption;
+@dynamic image;
+@dynamic likeCount;
+@dynamic commentCount;
+
+
++ (nonnull NSString *)parseClassName{
```

Comment:
This is not used (I think). If this is part of some inner workings of parse then it might actually deserve a comment? Not sure. It's confusing to me as a reader who is less familiar with Parse. Also should have a space before the opening brace.